src/routes/readerRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const { createReader, readReaders, getReaderById, updateReader, deleteReader } = require('../controllers/reader');
```

Comment:
This should be spread across multiple lines as in the other routers:
```suggestion
const {
  createReader,
  readReaders,
  getReaderById,
  updateReader,
  deleteReader,
} = require('../controllers/reader');
```